tests/e2e/validator.go

```diff
@@ -33,11 +33,11 @@ type validator struct {
 	index            int
 	moniker          string
 	mnemonic         string
-	keyInfo          keyring.Info
+	keyInfo          keyring.Record
 	privateKey       cryptotypes.PrivKey
 	consensusKey     privval.FilePVKey
 	consensusPrivKey cryptotypes.PrivKey
-	nodeKey          p2p.NodeKey
+	nodeKey          tmtypes.NodeKey
```

Comment:
I believe in the version of tendermint we're using here tmtypes.NodeKey isn't supported yet (til v0.35, we're using v0.34.20)

Reply:
yes, I'm surprised that this compiles. We depend on tendermint v0.34.20